dashboard-dev/install.bash

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+github_url="https://github.com/tue-robotics/dashboard.git"
+dest="$HOME/src/dashboard"
+
+tue-install-git "$github_url" "$dest"
```

Comment:
Dashboard is a catkin package. Either change the install type or remove package.xml and CMakeLists.txt from the dashboard repository.

Reply:
Agreed, I will remove it.